package.json

```diff
@@ -1,45 +1,11 @@
 {
-  "name": "nuxt-app",
+  "name": "unsightdotdev",
```

Comment:
I called the root package `unsightdotdev`, but feel free to change/suggest a better name @danielroe.